lib/application.json

```diff
@@ -55,14 +55,7 @@
     "name": "Julia Wild",
     "relationship": "Julia was my mentor at Cheadle Hulme High"
   }],
-  "offer": {
-    "course": {
-      "description": "Acorns Teaching School Alliance, Oxford Oakleaf Campus, Primary (salaried, full-time) PGCE, September 2020",
-      "provider_ucas_code": "2FR",
-      "course_ucas_code": "3CVK",
-      "location_ucas_code": "K"
-    },
-    "date": "2019-01-25",
+  "offer_conditions": {
```

Comment:
Ought this to be a plain `[]` rather than a hash with one key?

Reply:
Ah yes! 🤦‍♀ I will update this.